src/styles/stars.scss

```diff
@@ -0,0 +1,30 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &--4 {
+    display: flex;
+    align-items: center;
+  }
```

Comment:
The styles for the modifier `&--4` are redundant since they duplicate the styles declared for the `.stars` block. To follow the DRY principle, remove the redundant display and align-items properties here.